pytest.ini

```diff
@@ -1,5 +1,5 @@
 [pytest]
-addopts =
+xaddopts =
```

Comment:
Hey what is the difference between these 2 options?

Reply:
OOps. Shouldn't have committed that. Thanks for spotting it! I disable the pytest coverage locally because pycharm's debugger can't work if the pytest coverage is enabled.